### PR TITLE
Candidate form application

### PR DESCRIPTION
**Pull request by bhavyagor12**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -16,9 +16,10 @@
     "incremental": true,
     "baseUrl": ".",
     "paths": {
-      "@/*": ["./*"]
+      "@/*": ["./*"],
+      "react": [ "./node_modules/@types/react" ]
```
Comment (codingnuclei) [inline tsconfig.json]: Is this needed? see link below...

https://www.typescriptlang.org/tsconfig#types

**Review by codingnuclei — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (bhavyagor12) [reply tsconfig.json]: https://github.com/fakiolinho/react-loading/issues/66

@codingnuclei see i was facing this issue and saw stack overflow

Comment (bhavyagor12) [reply tsconfig.json]: @codingnuclei i dont understand what to do with the link can you help?

Comment (codingnuclei) [reply tsconfig.json]: Typescript by default includes all types in the @types folder in node_modules. So having a @type folder in ts paths should not be needed... we sure it is required here to fix the react loader? I just did a checkout of your pr and removed the tsconfig change and it seemed to work... can you check if it works for you without it?

Comment (bhavyagor12) [reply tsconfig.json]: image
see it wont let me build without that line
@codingnuclei

**Pull request merged**